build.sbt

```diff
@@ -484,6 +484,22 @@ lazy val jsoniterScala: ProjectMatrix = (projectMatrix in file("json/jsoniter"))
   )
   .dependsOn(core)
 
+lazy val zioJson: ProjectMatrix = (projectMatrix in file("json/zio"))
```

Comment:
should be added to the root project aggregates as well

Reply:
Ah that's good catch. Done!

Reply:
I realized that docs were not compiling due to the missing `dependsOn` to `zioJson`. I fixed that one as well so CI should pass now.